Automattic/jetpack
# Plugin Search: Add proof of concept

This adds a super super ugly/rough proof of concept of making suggestions when people search for functionality that Jetpack already provides.

We've seen that people with Jetpack installed and activated often search for Jetpack features (even by name) in the Plugins > Add New screen in wp-admin. This new module attempts to spot those searches, and provide an artificial search result that calls out that what they're looking for is in Jetpack, which they already have, and which is already active.

This code is _definitely_ not ready for prime time; it should hook into our existing search terms for settings search, should use better descriptions etc, needs to use real plugin data vs the hardcoded bits I used, and needs to make it really clear that this is an artificial search result, not something coming from the plugin directory directly.

### Testing Instructions

1. Switch to this branch
2. Plugins > Add New
3. Search for something that Jetpack provides (e.g "contact form")
4. See that the first result should be a Jetpack-powered card.

### Proposed changelog entry
- When people search plugins for features that Jetpack provides, give them hints.
